runtime/genesis.go

```diff
@@ -11,6 +12,7 @@ import (
 func parseGenesis(genesisJSONPath string) (g *genesis.GenesisState, err error) {
 	data, err := os.ReadFile(genesisJSONPath)
 	if err != nil {
+		err = fmt.Errorf("%w while reading %s", err, genesisJSONPath)
```

Comment:
this and below:
Without additional information I was not able to figure out which file contained a mistake.